Misc/NEWS.d/next/Library/2019-03-18-10-08-30.bpo-36337.QhJnXy.rst

```diff
@@ -0,0 +1,3 @@
+Use ssize_t and size_t instead of int for the send() function (see
+http://man7.org/linux/man-pages/man2/send.2.html (Linux) and
+https://nixdoc.net/man-pages/FreeBSD/man2/send.2.html (FreeBSD)).
```

Comment:
I propose:

Fix buffer overflow in :meth:`~socket.socket.send` and :meth:`~socket.socket.sendall` methods of :func:`socket.socket` for data larger than 2 GiB.